ServiceWebsite/ServiceWebsite.AcceptanceTests/Hooks/DataHooks.cs

```diff
@@ -100,8 +100,8 @@ private void UserShouldNotHaveAnswers(TestApiManager api)
         {
             var response = api.GetSuitabilityAnswers(_username);
             var answers = RequestHelper.Deserialise<List<PersonSuitabilityAnswerResponse>>(response.Content);
-            answers.Count.Should().Be(1, $"user with username '{_username}' has {answers.Count} previous answer(s) from other hearings saved");
-            if (answers.First().Answers.Count > 0)
+
+            if (answers != null && answers.First() != null && answers.First().Answers.Count > 0)
```

Comment:
could use `?` to check for null
`answers?.First()`

Reply:
done